packages/core/src/components/label/label.tsx

```diff
@@ -18,16 +21,29 @@ export class Label {
     ['danger', 'Gevaar']
   ]);
 
+  @Prop()
+  removable?: boolean;
+
+  @State()
+  hovered?: boolean;
+
+  @Event()
+  actionClick!: EventEmitter<MouseEvent>;
+
   render() {
     const status = this.status && Label.statusMap.get(this.status);
 
     return (
-      <span class={clsx('dso-label', { [`dso-label-${this.status}`]: this.status } )}>
+      <span class={clsx('dso-label', { [`dso-label-${this.status}`]: this.status, 'dso-label-small': this.small, 'dso-label-hover': this.hovered })}>
```

Comment:
De scope van het issue omschrijft het volgende:

>Het 'removable label' mag alleen 32px hoog zijn.

Maar ik zie geen beveiliging hierop terugkomen. Er iets niet wat mij weerhoudt van `<dso-label small removable>`

Reply:
Ik heb de check aangepast naar `'dso-label-compact': this.small && !this.removable`

Reply:
`this.compact` mag ik hopen.. Maar ik ga het zo zien ;-)

Reply:
Zeker!